medusa/indexers/tvmaze/tvmaze_api.py

```diff
@@ -371,7 +371,7 @@ def _parse_actors(self, tvmaze_id):
         for order, cur_actor in enumerate(actors.people):
             save_actor = Actor()
             save_actor['id'] = cur_actor.id
-            save_actor['image'] = cur_actor.image.get('original')
+            save_actor['image'] = cur_actor.image.get('original') if cur_actor.image else ''
```

Comment:
I know, but this is out of my control, as it's being serialized/deserialized by the lib. This is what it's giving me back.